server.go

```diff
@@ -632,8 +633,32 @@ func newServer(listenAddrs []net.Addr, chanDB *channeldb.DB, cc *chainControl,
 		return link.Bandwidth()
 	}
 
+	// Instantiate mission control with config from the sub server.
+	//
+	// TODO(joostjager): When we are further in the process of moving to sub
+	// servers, the mission control instance itself can be moved there too.
+	mcCfg := &routing.MissionControlConfig{
+		MinProbability:        routing.DefaultMinProbability,
+		PaymentAttemptPenalty: routing.DefaultPaymentAttemptPenalty,
+		PenaltyHalfLife:       routing.DefaultPenaltyHalfLife,
+	}
+
+	mcRefl := reflect.ValueOf(cfg.SubRPCServers.RouterRPC).Elem()
+	if mcRefl.NumField() != 0 {
+		mcCfg.MinProbability = mcRefl.FieldByName("MinProbability").
+			Float()
+
+		mcCfg.PaymentAttemptPenalty = lnwire.NewMSatFromSatoshis(
+			btcutil.Amount(mcRefl.FieldByName("AttemptCost").Int()),
+		)
+		mcCfg.PenaltyHalfLife = time.Hour *
```

Comment:
config says the value is in minutes, though this converts to hours? would be irrelevant tho if we can use `time.Duration` on the config